src/controller/CHIPCommissionableNodeController.cpp

```diff
@@ -48,6 +48,19 @@ CHIP_ERROR CommissionableNodeController::DiscoverCommissioners(Dnssd::DiscoveryF
     return mResolver->DiscoverCommissioners(discoveryFilter);
 }
 
+CHIP_ERROR CommissionableNodeController::StopDiscovery()
+{
+
+    if (mResolver == nullptr)
+    {
+        return AbstractDnssdDiscoveryController::StopDiscovery();
+    }
+    else
+    {
+        return mResolver->StopDiscovery();
+    }
```

Comment:
There is a CI check failure here: [https://github.com/project-chip/connectedhomeip/actions/runs/6191952094/job/16811156697?pr=29197](https://github.com/project-chip/connectedhomeip/actions/runs/6191952094/job/16811156697?pr=29197)

```
INFO TIDY ../../src/controller/CHIPCommissionableNodeController.cpp: /Users/runner/work/connectedhomeip/connectedhomeip/out/default/../../src/controller/CHIPCommissionableNodeController.cpp:58:5: error: do not use 'else' after 'return' [readability-else-after-return,-warnings-as-errors]
2831 58 | else
2832 | ^~~~
2833 59 | {
2834 | ~
2835 60 | return mResolver->StopDiscovery();
2836 | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2837 61 | }
2838 | ~
```